### DISCUSSION TITLE
Making `mini.clue` work with `Comment.nvim`

### DISCUSSION THREAD

**Discussion by simonmandlik** [nvim-mini/mini.nvim#472] [Q&A] [answered]
How exactly do I make these two plugins work together? I have the same problem described [here](https://github.com/echasnovski/mini.nvim/issues/430#issuecomment-1664422151), but I don't know where to call `ensure_buf_triggers()`.

Also, I checked `Comment.nvim` implementation, and it seems to me that it doesn't create any buffer-attached mappings. Also, I have mini.clue maps on top of `:nmap g`:

```
:nmap g
n  g`          *@<Lua 189: ~/.local/share/nvim/lazy/mini.clue/lua/mini/clue.lua:1255>
                 Query keys after "g`"
n  g'          *@<Lua 191: ~/.local/share/nvim/lazy/mini.clue/lua/mini/clue.lua:1255>
                 Query keys after "g'"
n  g           *@<Lua 198: ~/.local/share/nvim/lazy/mini.clue/lua/mini/clue.lua:1255>
                 Query keys after "g"
```

**Comment by echasnovski** [comment, marked as answer]
> How exactly do I make these two plugins work together? I have the same problem described [here](https://github.com/echasnovski/mini.nvim/issues/430#issuecomment-1664422151), but I don't know where to call `ensure_buf_triggers()`.
> 
> Also, I checked `Comment.nvim` implementation, and it seems to me that it doesn't create any buffer-attached mappings. Also, I have mini.clue maps on top of `:nmap g`:
> 
> ```
> :nmap g
> n  g`          *@<Lua 189: ~/.local/share/nvim/lazy/mini.clue/lua/mini/clue.lua:1255>
>                  Query keys after "g`"
> n  g'          *@<Lua 191: ~/.local/share/nvim/lazy/mini.clue/lua/mini/clue.lua:1255>
>                  Query keys after "g'"
> n  g           *@<Lua 198: ~/.local/share/nvim/lazy/mini.clue/lua/mini/clue.lua:1255>
>                  Query keys after "g"
> ```

What is the reason having all `g`, `g'`, and ``g``` as triggers? When you press `g`, it already should start a key query process. Having other ones might interfere with correct reproducing of key presses. Try either removing `g` or the other two from triggers.

**Reply by simonmandlik** [reply]
Yes, removing them works. Thanks!
